src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.scss

```diff
@@ -0,0 +1,90 @@
+@import '../../../styles/variables';
+
+.box-ui-activity-feed {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    position: relative;
+
+    .box-ui-activity-feed-items-container {
+        align-items: center;
+        display: flex;
+        flex: 1 1;
+        flex-direction: column;
+        overflow-y: auto;
+        overflow-x: hidden;
+        position: relative;
+
+        .box-ui-version-item {
+            &:first-child .box-ui-version {
+                margin-top: 0;
+            }
+        }
+
+        .box-ui-activity-feed-task,
+        .box-ui-activity-feed-comment {
+            border-bottom: 1px solid $ffive;
+            padding-bottom: 8px;
+
+            &:first-child .box-ui-comment {
+                margin-top: 0;
+            }
+        }
+
+        .box-ui-activity-feed-task {
+            padding-bottom: 28px;
+        }
+    }
+
+    .box-ui-activity-feed-active-state {
+        bottom: 0;
+        left: 0;
+        margin: 9px 20px;
+        position: absolute;
+        right: 0;
+        top: 0;
+
+        > li:last-child {
+            margin-bottom: 10px;
+        }
+    }
+
+    .box-ui-activity-feed-empty-state {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+
+        .empty-state-cta {
```

Comment:
All classes should be pre-fixed, otherwise there will be quite a few conflicts with webapp, since they will also have old activity feed classes

Reply:
That was a doozy but, but I did not add for ALL classes. Only those that were A) referenced in a JS file as a class name and B) Had a corresponding entry in a SASS file. 

There are certain instances (ie. `.flyout-wrapper`, `.overlay-wrapper` etc.) that do not have a corresponding assignment in JS. I have left those as is as to not cause conflict. Let me know if I should remove those.